games-roguelike/hengband/hengband-1.6.2-r1.ebuild

```diff
@@ -2,7 +2,7 @@
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=5
-inherit eutils autotools games
+inherit eutils autotools
 
 DESCRIPTION="An Angband variant, with a Japanese/fantasy theme"
 HOMEPAGE="http://hengband.sourceforge.jp/en/"
```

Comment:
Down below, sort to the canonical 'block structure':
LICENSE
SLOT
KEYWORDS
IUSE